server/server.js

```diff
@@ -14,16 +14,52 @@ app.use(express.urlencoded({extended: true}));
 app.use(morgan('dev'))
 app.use(cors())
 
-app.get('*',(req,res) =>{
-  res.sendFile(path.join(__dirname+'/../public/index.html'))
+app.use('/:listingId', (req, res, next) => {
+  if (req.params.listingId > 99 || req.params.listingId < 0) {
+    res.sendStatus(404)
+  }
+  next()
+})
+app.use('/:listingId', express.static(__dirname + '../../public'));
+app.use(express.json());
+
+
+
+app.get('/api/host/:listingId', (req, res) => {
+  res.redirect(`http://ec2-3-12-169-208.us-east-2.compute.amazonaws.com:2000/api/host/${req.params.listingId}`)
+})
+
+
+app.get('/api/reservation/:listingId', (req, res) => {
+  res.redirect(`http://ec2-52-14-154-112.us-east-2.compute.amazonaws.com/api/reservation/${req.params.listingId}`)
+})
+
+app.get('/api/location/:listingId', (req, res) => {
+  res.redirect(`http://ec2-3-12-169-208.us-east-2.compute.amazonaws.com:2001/api/location/${req.params.listingId}`)
 })
 
-app.use((req,res,next) =>{
-  res.header('Access-Control-Allow-Origin', '*');
-  res.header('Access-Control-Headers','Origin','X-Requested-With','Content-Type','Accept')
-  next();
+app.get('/:listingId/airbrb_home.png', (req, res) => {
```

Comment:
All these redirects are unnecessary now since we hardcoded our fetch requests?

Reply:
yeah i dont think they are necesary anymore, i will delete them once everything is finalized. just keepin them just in case